### DISCUSSION TITLE
[ZK Stack] - <Info about Decentralized sequencers for zk stack>

### DISCUSSION THREAD

**Discussion by namn-grg** [zkSync-Community-Hub/zksync-developers#215] [ZK Stack] [answered]
### Module Affected

- [ ] System contracts (era-system-contracts)
- [ ] Smart contracts (era-contracts)
- [ ] zkEVM (era-zk_evm)
- [ ] Server (zksync-era)
- [ ] Prover (era-boojum)
- [ ] Fee withdrawer (era-fee-withdrawer)
- [ ] Block Explorer (clock-explorer)
- [ ] Portal - Wallet + Bridge (dapp-portal)
- [X] Other (please specify below)

### Rust Version

any

### Operating System

- [ ] Ubuntu
- [ ] MacOS
- [ ] Windows
- [X] Other (Please specify below)

### Issue Description

Info about Decentralized sequencers for zk stack, any writeups, proposals etc.

### Expected Behavior

.

### Current Behavior

.

### Prior Research

- [X] Yes, I searched for similar issues and did not find any matching ones.

**Comment by MexicanAce** [comment]
Hi @zk-lumi,

Thoughts?

**Comment by githubdoramon** [comment, marked as answer]
@namn-grg we are working on a new version of our sequencer that will indeed be decentralized. There are no write-ups yet, but expect some news in Q1 2024 about it. The goal is to indeed provide a new module for ZK Stack allowing hyperchains to have a decentralized set of validators.

**Comment by varnerrobertv** [comment]
For information about Decentralized Sequencers for zkSync, you can refer to the zkSync documentation, zkSync whitepaper, or explore discussions and proposals on zkSync's official GitHub repository or forums like Ethereum Research. Additionally, you may find relevant insights from zkSync community meetups, developer conferences, or presentations by the zkSync team.

**Comment by longbipubacr** [comment]
Decentralized sequencers for zk stack typically involve distributed systems that collectively generate and finalize zk-rollup blocks. They aim to enhance scalability and decentralization while maintaining security and efficiency. Key components often include verifiers, aggregators, and relayers. Research into decentralized sequencers for zk-stack can be found in academic papers, protocol specifications, and community discussions. Additionally, zkSync, Optimism, and Arbitrum are prominent projects exploring this concept.

**Comment by fitzgibbonsdustin** [comment]
For information about decentralized sequencers for the zkSync stack, including writeups, proposals, and discussions, you may want to explore the zkSync GitHub repository, zkSync documentation, and related forums such as zkSync's Discord channel or community forums. Additionally, you can search for relevant articles and research papers on decentralized sequencers in the context of zk rollups and layer 2 scaling solutions.

**Comment by yamyv** [comment]
For information on decentralized sequencers for the zk stack, including write-ups and proposals, please refer to the zkSync documentation, zkSync GitHub repository, and relevant community forums such as zkSync Discord or zkSync GitHub Discussions.

**Comment by shipefred** [comment]
Module Affected: Prover (era-boojum)
Operating System: Ubuntu, MacOS, Windows
Issue Description: Seeking information about decentralized sequencers for zk stack, including writeups, proposals, etc.
Expected Behavior: -
Current Behavior: -
Repository Link: N/A
Additional Details: N/A
Prior Research: Yes

**Comment by davehutchinsv** [comment]
The user is seeking information about decentralized sequencers for the zk stack, including writeups and proposals. They have not provided any expected or current behavior, repository link, or additional details. The user has stated that they have searched for similar issues and did not find any matching ones. Further investigation and research may be needed to provide a solution or answer to this question.
